docs/asciidoc/static/offline-plugins.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[offline-plugins]]
+=== Offline Plugin Management
+
+The Logstash <<working-with-plugins,plugin manager>> was introduced in the 1.5 release. This section discusses setting up
+local repositories of plugins for use on systems without access to the Internet.
+
+The procedures in this section require a machine running Logstash that has access to a public or private Rubygems server.
+See the <<private-rubygem,Private Gem Repositories>> section for information on setting up your own private Rubygems 
+server.
+
+[float]
+=== Using the Offline Pack
+
+Working with offline plugins requires you to create an _offline pack_, which is a compressed file that contains all of the
+plugins your offline Logstash installation requires, along with the dependencies for those plugins. You create the
+offline pack with the `bin/plugin pack` subcommand.
+
+When you run the `bin/plugin pack` subcommand, Logstash creates a compressed bundle that contains all of the currently
+installed plugins and the dependencies for those plugins. By default, the compressed bundle is a GZipped TAR file when you 
+run the `bin/plugin pack` subcommand on a UNIX machine. By default, when you run the `bin/plugin pack` subcommand on a 
+Windows machine, the compressed bundle is a ZIP file. See <<managing-packs,Managing Plugin Packs>> for details on changing 
+these default behaviors.
```

Comment:
+1 on this, but keep in mind this is only in the first request where the internal caches are rebuild.